tests/utils/test_logging.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for the logging decorator"""
+
+import logging
```

Comment:
Check how imports should be ordered in Python. See [here](https://www.python.org/dev/peps/pep-0008/#imports) for the relevant section in PEP8.